api/src/endpoints/enzymeDb.js

```diff
@@ -0,0 +1,70 @@
+import express from 'express';
+import { getCompound, getEc, getReaction, getEnzymes } from 'enzymeDb/index';
+
+const enzymeDbRoutes = express.Router();
+
+enzymeDbRoutes.get('/compounds/:id', async (req, res) => {
+  const { id } = req.params;
+
+  try {
+    const compound = await getCompound(id);
+    if (compound) {
+      res.json(compound);
+    } else {
+      res.sendStatus(404);
+    }
+  } catch (e) {
+    console.error(e.message);
+    res.sendStatus(400);
```

Comment:
Seems strange to me to return 400 Bad Request here? The only thing passed in as id and if nothing is found using using that id it should be a 404 and if there are multiple ids for the id the data is bad, not the request? Seems more natural to me to return 500

Reply:
Really good point here @perjo. I think we have been a bit sloppy with the error codes. Do you think it's alright if we address this in a new issue? We are using `400` in some other places where changes could be needed too.

<img width="645" alt="image" src="https://user-images.githubusercontent.com/423498/173594676-2f2c322d-6ec0-4b24-931c-349b520e0779.png">

Reply:
I think `404` and `500` are really good and sensible suggestions. If we are changing though, I would still like to evaluate more possibilities of code (implementation) changes or alternative status codes. So I would prefer if we can have a bit more discussion about this before changing.

The main concern for me is to do what's convenient for collaborators. In general I would be thinking about the PR size and complexity. In this case since this issue is a blocker for a few others I think that's another good reason to continue this in another issue. Having said that, if the reviewers still think it's better to handle this here, or at least add the changes suggested by Pelle, that would be fine with me as well. Please let me know!

Reply:
Apologies again for not double checking earlier. I misunderstood the original situation and thought we only had `400` and that @perjo suggested to change the code to include both `404` and `500`. I took a look now and it seems that I had forgotten that we had already implemented the `404` bit, so since the only suggestion was to change `400` to `500`, that seems trivial enough to be included here.

I have added [a commit](https://github.com/MetabolicAtlas/MetabolicAtlas/pull/1030/commits/9ba0ffa31f8fe7e7d90882a6ed7c6f51cef6a016) for this now. It also includes a change for the `enzymes` endpoint. `404` does not make sense there since if there are 0 matches, it still returns a valid response that looks like `{"enzymes":[],"totalCount":"0"}`.